pkg/dbt_cloud/job.go

```diff
@@ -9,15 +9,15 @@ import (
 )
 
 type JobTrigger struct {
-	Github_Webhook     bool `json:"github_webhook"`
```

Comment:
Did you update all of those by hand or used some tool for it? I have been writing new code with CamlCase but didn't touch most of the code that was written before I got involved.

Reply:
I used IntelliJ (GoLand) to do the rename, so it updates all references as well.